### DISCUSSION TITLE
http接口报错

### DISCUSSION THREAD

**Discussion by Parrrdo** [hiroi-sora/Umi-OCR#459] [General]
http接口报错
requests.exceptions.invalidschema : no connection adapters were found for 'http://127.0.0.1:1224/api/ocr '
这是什么原因呢？

**Comment by hiroi-sora** [comment]
1. 如果用 [HTTP手册](https://github.com/hiroi-sora/Umi-OCR/blob/main/docs/README_HTTP.md) 中的示例代码，可以正常调用吗？
2. 如果示例正常，你的代码出现问题，可以把代码发上来给我看看。

**Reply by Parrrdo** [reply]
代码没问题，现在可以正常调用了。估计是当时的配置环境有问题。另外，做为小白，想问问大佬，调用必须要启动umi-ocr挂后台吗？可不可以直接import啊？

**Reply by hiroi-sora** [reply]
如果你想用Python调用OCR，那么 [PaddleOCR-json](https://github.com/hiroi-sora/PaddleOCR-json) 或者 RapidOCR-json 更简单省事。它们就是Umi所采用的引擎库，提供了Python接口。
